gen-readme.py

```diff
@@ -11,6 +12,17 @@ def get_module_docstring(filepath):
         docstring = None
     return docstring
 
+def get_mojo_implementation_tag(filepath):
+    with open(filepath, "r") as source:
+        root = ast.parse(source.read())
+        for node in (n for n in ast.walk(root) if isinstance(n, ast.Assign)):
+            if isinstance(node.targets[0], ast.Name) and node.targets[0].id == '_mojo':
+                if isinstance(node.value, ast.NameConstant) and node.value.value:
+                    return ":hammer:"
```

Comment:
@JasminePParekh we need to explain what the EMOJI means. Can you pls put a tiny note in README to explain it?